evm-bridging/src/interfaces/ICrossVMBridgeFulfillment.sol

```diff
@@ -0,0 +1,6 @@
+pragma solidity 0.8.24;
+
+interface ICrossVMBridgeFulfillment {
```

Comment:
Interface conformance expected to be needed to onboard a custom contract in upcoming upgraded bridge